src/AVFoundation/Enums.cs

```diff
@@ -930,12 +930,13 @@ public enum AVVideoApertureMode {
 		EncodedPixels = 2,
 	}
 
+	// Enum values are present in header but missing rdar filed at: https://bugreport.apple.com/web/?problemID=34184435
```

Comment:
This is not what I asked, comment the whole type (not just the fields) and API using it. We're not sure how this will be fixed so exposing metadata (that can't be used) is not helpful and could block us from a better way to implement this.